tests/e2e-pw/specs/merchant/merchant-payment-gateways-confirmation.spec.ts

```diff
@@ -8,21 +8,18 @@ import { test, expect, Page } from '@playwright/test';
  */
 import { useMerchant } from '../../utils/helpers';
 
-// Skipping the test for now as it is flaky on GH action runs. See #8875.
-test.skip( 'payment gateways disable confirmation', () => {
+test.describe( 'payment gateways disable confirmation', () => {
```

Comment:
💭 I'm unsure if it is in scope for this PR or project, but have you considered adding `@critical` tags to these tests as described in this RFC for measuring critical flow e2e coverage paJDYF-dVT-p2 (authored by @alopezari)?

```suggestion
test.describe( 'payment gateways disable confirmation', {
	tag: '@critical',
},
() => {
```

Mind you, this is not currently documented in the [Playwright E2E README](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e-pw/README.md#L1), but we have utilised `@critical` tagging this for some e2e tests e.g. https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e-pw/specs/merchant/merchant-admin-account-balance.spec.ts.

I'm also assuming that this is a critical flow, which may be incorrect. 😅

Reply:
Thanks for flagging this. I see you also commented on the project thread about it. We can move the discussion forward there. That said, I checked and I don't believe these tests are part of the current [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows).